### DISCUSSION TITLE
Analog Trigger Support on DS4 / Xbox One

### DISCUSSION THREAD

**Discussion by gdstcyg** [cathery/sys-con#207] [Feature requests]
Will the analog triggers be supported in future versions?
It could be truly handy when playing racing games like GRID Autosports...
Thanks in advance XD

**Comment by tuxinal** [comment]
there is no analog trigger support for switch, neither the joycons nor the pro controller have analog triggers

**Comment by mathieu4202** [comment]
GameCube controller analog triggers work with the Switch. 
It should be possible to do the same with all controllers that have analog triggers.

**Comment by tuxinal** [comment]
they do work but it isn't an analog signal its a digital signal there is no in-between state it's either pressed or not pressed, that's exactly what this program does right now

**Comment by mathieu4202** [comment]
I do not agree. For example, they are natively supported in Grid autosport in analog.
And there are also other compatible games.
